update-script.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+"""
+Performs the three git commit required to do a release. See help output for more details.
+"""
+
+import subprocess
+import re
+import sys
+
+
+def cmd(command: list[str]) -> str:
+    print('$ %s', ' '.join(command))
```

Comment:
Should we use f string here

Reply:
alternatively, if you want to avoid nesting quotes, you could just concat them
```python
print(f'$ {" ".join(command)}')
# vs
print('$ ' + ' '.join(command))
```

Reply:
Done!